src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -102,10 +108,112 @@ def validate_role(self, role):
         return role
 
 
+class OrganizationMemberIndexQueryParamSerializer(serializers.Serializer):
```

Comment:
Note that the endpoint doesn't actually use this query param serializer for now, purely for documentation purposes. I'll update the endpoint to use it (along with any extra tests needed) separately.